src/components/datatrust/StartListingButton.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="button-container">
+    <button class="button" @click.prevent="startListing">Start Listing</button>
+  </div>
+</template>
+
+<script lang="ts">
+  import Vue from 'vue'
+  import FlashesModule from '../../modules/FlashesModule'
+  import { getModule } from 'vuex-module-decorators'
+  import Flash from '../../models/Flash'
+  import { FlashType } from '../../models/Flash'
+
+  export default Vue.extend({
+    methods: {
+      startListing() {
+        const flashesModule = getModule(FlashesModule, this.$store)
+        const flash = new Flash('Starting Listing', FlashType.info)
```

Comment:
let's not use hard coded strings. pretty sure i started a `constants` file

Reply:
Probably worth a ticket to track the rest of these down and 'retire' them like unlicensed, off-world replicants.  Godspeed Rutger Hauer.

Reply:
<sound of a pistol racking>